vulture/lines.py

```diff
@@ -1,12 +1,11 @@
 import ast
-from functools import wraps
 import sys
 
 TRAVERSABLE_FIELDS = {
-        ast.ClassDef: ('body', 'decorator_list'),
+        ast.ClassDef: ('decorator_list', 'body', ),
         ast.ExceptHandler: ('body',),
         ast.For: ('body', 'orelse'),
-        ast.FunctionDef: ('body', 'decorator_list'),
+        ast.FunctionDef: ('decorator_list', 'body',),
         ast.If: ('body', 'orelse'),
         ast.Module: ('body',),
         ast.While: ('body', 'orelse'),
```

Comment:
Since `ast.FunctionDef.returns` returns `None`, it messes up with the 'fields'. We probably won't need it.